cluster-provision/gocli/cmd/rm.go

```diff
@@ -39,12 +39,12 @@ func rm(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	var dnsmasq *types.Container
+	var dnsmasq []*types.Container
 nodnsmasq:
 	for i, c := range containers {
 		for _, name := range c.Names {
 			if strings.HasSuffix(name, "dnsmasq") {
-				dnsmasq = &containers[i]
+				dnsmasq = append(dnsmasq, &containers[i],)
```

Comment:
wouldn't it readd again and again the dnsmasq container because the continue in next line ?
(iterating on the containers again, trying to delete already deleted ones, and adding the already added ones to the list)
on the other hand, it would have happen also before if so, so i guess i am missing something